.github/workflows/release-genesis-creator.yml

```
@@ -0,0 +1,69 @@
+name: 'Publish genesis-creator binaries'
+on:
+  # This release action always requires a tag to publish a release.
+  # This job will run when a genesis-creator/$VERSION
+  # tag is created.
+  push:
+    tags:
+      - genesis-creator/*.*.*
+
+jobs:
+  build:
+    name: build ${{ matrix.platform }}
+    strategy:
+      fail-fast: true
+      matrix:
+        platform: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Support longpaths on Windows
+        if: matrix.platform == 'windows-latest'
+        run: git config --system core.longpaths true
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@1.85
+      - name: Build
+        run: cargo build --release --package genesis-creator
```

I think we should enforce the dependencies from the lock file:
```suggestion
        run: cargo build --locked --release --package genesis-creator
```